bedrock/press/templates/press/speaker-request.html

```diff
@@ -1,274 +1,279 @@
+      </div>
+
+      <li>
+        {{ form.sr_guest_speaker1.errors }}
+        <label for="sr_guest_speaker1">{{ _('Guest Speaker Choice #1') }}</label>
+        {{ form.sr_guest_speaker1 }}
+      </li>
+
+      <li>
+        {{ form.sr_guest_speaker2.errors }}
+        <label for="sr_guest_speaker2">{{ _('Guest Speaker Choice #2') }}</label>
+        {{ form.sr_guest_speaker2 }}
+      </li>
+    </ol>
+  </fieldset>
+
+  <fieldset>
+    <legend>{{ _('Contact Information') }}</legend>
+
+    <ol>
+      <li>
+        {{ form.sr_contact_name.errors }}
+        <label for="sr_contact_name">{{ _('Name') }}</label>
+        {{ form.sr_contact_name }}
+      </li>
+
+      <li>
+        {{ form.sr_contact_title.errors }}
+        <label for="sr_contact_title">{{ _('Title') }}</label>
+        {{ form.sr_contact_title }}
+      </li>
+
+      <li>
+        {{ form.sr_contact_company.errors }}
+        <label for="sr_contact_company">{{ _('Company') }}</label>
+        {{ form.sr_contact_company }}
+      </li>
+
+      <li>
+        {{ form.sr_contact_phone.errors }}
+        <label for="sr_contact_phone">{{ _('Phone') }}</label>
+        {{ form.sr_contact_phone }}
+      </li>
+
+      {{ form.office_fax }}
+
+      <li>
+        {{ form.sr_contact_email.errors }}
+        <label for="sr_contact_email">{{ _('Email') }}</label>
+        {{ form.sr_contact_email }}
+      </li>
+
+      <li>
+        {{ form.sr_contact_company_url.errors }}
+        <label for="sr_contact_company_url">{{ _('Company URL') }}</label>
+        {{ form.sr_contact_company_url }}
+      </li>
+    </ol>
+  </fieldset>
+
+  <fieldset>
+    <legend>{{ _('Event Details') }}</legend>
+
+    <ol>
+      <li>
+        {{ form.sr_event_venue.errors }}
+        <label for="sr_event_venue">{{ _('Venue') }}</label>
+        {{ form.sr_event_venue }}
+      </li>
+
+      <li>
+        {{ form.sr_event_theme.errors }}
+        <label for="sr_event_theme">{{ _('Theme') }}</label>
+        {{ form.sr_event_theme }}
+      </li>
+
+      <li>
+        {{ form.sr_event_goal.errors }}
+        <label for="sr_event_goal">{{ _('Goal') }}</label>
+        {{ form.sr_event_goal }}
+      </li>
+
+      <li>
+        {{ form.sr_event_format.errors }}
+        <label for="sr_event_format">{{ _('Format') }}</label>
+        {{ form.sr_event_format }}
+      </li>
+
+      <li>
+        {{ form.sr_event_audience_size.errors }}
+        <label for="sr_event_audience_size">{{ _('Expected Audience Size') }}</label>
+        {{ form.sr_event_audience_size }}
+      </li>
+
+      <li>
+        {{ form.sr_event_audience_demographics.errors }}
+        <label for="sr_event_audience_demographics">{{ _('Audience Demographics') }}</label>
+        {{ form.sr_event_audience_demographics }}
+      </li>
+
+      <li>
+        {{ form.sr_event_speakers_confirmed.errors }}
+        <label for="sr_event_speakers_confirmed">{{ _('Confirmed Speakers') }}</label>
+        {{ form.sr_event_speakers_confirmed }}
```

Comment:
The text areas break out of the viewport at small screen sizes:

![image](https://user-images.githubusercontent.com/400117/69134608-ad397580-0aaf-11ea-8dcc-1e8bdb1e6421.png)

Reply:
I have hit a wall with this one. I am totally lost on how to make these text areas behave & stay within their containers. 🤦‍♀

Reply:
@amychurchwell instead of using the `cols` and `rows` attributes in the HTML, you may want to try removing those and using values in CSS instead to make them responsive. Something like `width: 100%; min-height: 200px;` might work.

Reply:
Looks like you can do this using the following in `forms.py`:

```python
widget=forms.Textarea(
    attrs={
        'rows': '',
        'cols': '',
    }
),
```